content/_includes/quotes.macros.njk

```diff
@@ -121,10 +119,15 @@ params:
 
     {%- if slugs -%}
       {%- set content -%}
-        {%- for item in slugs -%}
-          {%- set quote = data | findData(false, {'slug': item}) -%}
-          {{- blockquote(quote, 'grid') if quote else none -}}
-        {%- endfor -%}
+      {%- for item in slugs -%}
+        {%- set quote = data | findData(false, {'slug': item}) -%}
+        {{- blockquote(quote, 'grid') if quote else none -}}
+        {{- utility.link_if(
+            content= 'See work sample »',
+            url=url,
+            attrs={'class': 'offset-more'}
+          ) if url and loop.last else none -}}
```

Comment:
@dvdherron @SondraE I wonder if we might want to make the "See work sample »" label editable so it can be used to link to something other than a work sample if we choose to use the quote macro elsewhere? The work page has the link label "Read ClientName Case Study »" as part of the template, but that is specific to the work page as opposed to a macro. This isn't of high importance at this moment since we don't use this for anything else.

Reply:
@stacy clean this up in [f5c12c6](https://github.com/oddbird/oddleventy/pull/193/commits/f5c12c662a956f4b5e15143f251ada8df945d6fd). The link text is now represented by the `action` variable.

Reply:
@dvdherron beautiful!